pyqldb/errors/__init__.py

```diff
@@ -18,6 +18,7 @@
     ReadTimeoutError, EndpointConnectionError, ConnectionClosedError, ConnectTimeoutError
 )
 
+
```

Comment:
Extra newline? Meant to be here?

Reply:
Yes, [PEP](https://legacy.python.org/dev/peps/pep-0008/#blank-lines) says to surround classes with two blank lines. For future PRs, I will minimize these non-related changes (put them in another PR)